src/components/Wizard/CreateImageWizard.js

```diff
@@ -93,7 +93,10 @@ const CreateImageWizard = (props) => {
 
     let uploadSettings;
     if (formValues?.image?.isUpload) {
-      if (formValues?.image?.type === "ami") {
+      if (
+        formValues?.image?.type === "ami" ||
+        formValues?.image.type === "edge-ami"
```

Comment:
Looking at our integration tests, it doesn't look like we upload directly to aws for this image type.
https://github.com/osbuild/osbuild-composer/blob/dc389eaa71936790349577b81281fae7ff4c5d9e/test/cases/ostree-ami-image.sh#L637